docs/user-guide/longitudinal-profile.md

```diff
@@ -0,0 +1,87 @@
+# Longitudinal Profile
+
+*******************
+
+Given a DEM or a bathymetric layer as a source and a provided path on it, this tool allows to calculate the **Longitudinal Profile** and display it within an interactive chart.
+
+!!! note
+    The **Longitudinal Profile** is not active by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed.
+    The plugin works only if the [WPS process](gs-wps-longitudinal-profile-2.24-SNAPSHOT.zip) is configured on the [GeoServer](http://geoserver.org/).
```

Comment:
```suggestion
    The plugin works only if the **Longitudinal Profile WPS process** (available as a community extension) is properly installed in [GeoServer](http://geoserver.org/).
```

Reply:
@ElenaGallo we cannot link the official GS documentation because the long profile process has not been merged yet as a community extension. We have to wait for this and finally provide an update to this doc (create a dedicate issue please)

Reply:
New issue here: https://github.com/geosolutions-it/MapStore2/issues/9299